_doc-build-windows/action.yml

```
@@ -320,10 +343,12 @@ runs:
         BUILD_BACKEND: ${{ steps.determine-environment-variables.outputs.BUILD_BACKEND }}
       run: |
         .venv\Scripts\Activate.ps1
-        if ("$env:BUILD_BACKEND" -eq 'poetry' ) {
-          poetry install
+        if ($env:BUILD_BACKEND -eq "poetry") {
+            poetry install
+        } elseif ($env:BUILD_BACKEND -eq "uv") {
+            uv pip install .
```

```suggestion
        if ($env:BUILD_BACKEND -eq "poetry") {
          poetry install
        } elseif ($env:BUILD_BACKEND -eq "uv") {
          uv pip install .
```
To match files formatting